packages/bundlers/default/src/DefaultBundler.js

```diff
@@ -451,6 +400,7 @@ export default (new Bundler({
 
     // Remove assets that are duplicated between shared bundles.
     deduplicate(bundleGraph);
+    internalizeReachableAsyncDependencies(bundleGraph);
```

Comment:
Any perf numbers to report on this extra step?

Reply:
(see below)